packages/ripple-keypairs/src/signing-schemes/ed25519/index.ts

```diff
@@ -0,0 +1,56 @@
+import { ed25519 as nobleEd25519 } from '@noble/curves/ed25519'
+import { bytesToHex } from '@xrplf/isomorphic/utils'
+
+import type { HexString, SigningScheme } from '../../types'
+import assert from '../../utils/assert'
+import Sha512 from '../../utils/Sha512'
+
+const ED_PREFIX = 'ED'
+
+const ed25519: SigningScheme = {
+  deriveKeypair(entropy: Uint8Array): {
```

Comment:
Should this also take an `options` param and just not do anything with it, for uniformity/ease of use?

Reply:
I dont see a need to as SigningScheme.deriveKeypair marks it as an optional param anyway.  Also I believe our lint config frown on unused end params